troposphere/batch.py

```diff
@@ -10,13 +10,13 @@ class ComputeResources(AWSProperty):
         "SecurityGroupIds": ([basestring], True),
         "BidPercentage": (positive_integer, False),
         "Type": (basestring, True),
-        "Subnets": ([basestring], True),
+        "Subnets": (list, True),
         "MinvCpus": (positive_integer, True),
         "ImageId": (basestring, False),
         "InstanceRole": (basestring, True),
         "InstanceTypes": ([basestring], True),
         "Ec2KeyPair": (basestring, False),
-        "Tags": ([basestring], False),
+        "Tags": (dict, False),
```

Comment:
The [documentation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-batch-computeenvironment-computeresources.html#cfn-batch-computeenvironment-computeresources-tags) is all over the map on this one as it is referring to the Tags as being normal [Tags](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html). I'll have to follow up with my AWS contacts to understand why there is a discrepancy.

Reply:
I understand. It took me a while to figure out how AWS Batch tags work, but when using normal Tags CloudFormation gives the following message:
```
Property validation failure: [Value of property {/ComputeResources/Tags} does not match type {Map}]
```
which is very weird when comparing with the docs.

Reply:
Do you have any news about this?